proto/node/akash/market/v1/reclamation.proto

```
@@ -0,0 +1,39 @@
+
+  // started_at is the block height at which reclamation was initiated.
+  // Zero means reclamation has not been started yet.
+  int64 started_at = 2 [
+    (gogoproto.jsontag)  = "started_at",
+    (gogoproto.moretags) = "yaml:\"started_at\""
+  ];
+
+  // deadline is the unix timestamp at which the reclamation window expires.
+  // Zero means reclamation has not been started yet.
+  int64 deadline = 3 [
+    (gogoproto.jsontag)  = "deadline",
```

Buf (1.68.4)

[error] cannot find `gogoproto.jsontag` in this scope

(COMPILE)